live-examples/css-examples/transforms/meta.json

```diff
@@ -7,175 +7,196 @@
             "title": "CSS Demo: backface-visibility",
             "type": "css"
         },
-        "perspective": {
-            "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
-            "exampleCode": "./live-examples/css-examples/transforms/perspective.html",
-            "fileName": "perspective.html",
-            "title": "CSS Demo: perspective",
+        "functionMatrix": {
+            "exampleCode": "./live-examples/css-examples/transforms/function-matrix.html",
+            "fileName": "function-matrix.html",
+            "title": "CSS Demo: matrix()",
+            "type": "css"
+        },
+        "functionMatrix3d": {
+            "exampleCode": "./live-examples/css-examples/transforms/function-matrix3d.html",
+            "fileName": "function-matrix3d.html",
+            "title": "CSS Demo: matrix3d()",
             "type": "css"
         },
-        "perspectiveFunction": {
+        "functionPerspective": {
             "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
             "exampleCode": "./live-examples/css-examples/transforms/function-perspective.html",
             "fileName": "function-perspective.html",
             "title": "CSS Demo: perspective()",
             "type": "css"
         },
-        "perspectiveOrigin": {
-            "cssExampleSrc": "./live-examples/css-examples/transforms/perspective-origin.css",
-            "exampleCode": "./live-examples/css-examples/transforms/perspective-origin.html",
-            "fileName": "perspective-origin.html",
-            "title": "CSS Demo: perspective-origin",
-            "type": "css"
-        },
-        "rotate": {
+        "functionRotate": {
             "exampleCode": "./live-examples/css-examples/transforms/function-rotate.html",
             "fileName": "function-rotate.html",
             "title": "CSS Demo: rotate()",
             "type": "css"
         },
-        "rotate3d": {
+        "functionRotate3d": {
```

Comment:
I think the rename makes sense, but are there any side-effects of renaming these keys?

Reply:
No, those keys don't matter at all, as long as they are unique in the same file.

Reply:
I had to check again how we're embedding in the content repo, for example for `border` CSS prop we're using:

```
{{EmbedInteractiveExample("pages/css/border.html")}}
```

My concern was how we reference them and if this renaming had any impact, so I guess this is fine :)

Reply:
Currently, the path is built by always placing "pages/" on the front, then the value of property "type" is included, then "/" and then the value of fileName property. So it also matters that file names don't collide in the same type.